yb-voyager/cmd/getDataMigrationReportCommand.go

```diff
@@ -316,42 +332,54 @@ func updateImportedEventsCountsInTheRow(sourceDBType string, row *rowData, table
 	}
 	if importerRole == SOURCE_REPLICA_DB_IMPORTER_ROLE {
 		var err error
-		tblName := tableNameTup.ForKey()
-		tableNameTup, err = nameRegistryForSourceReplicaRole.LookupTableName(tblName)
-		if err != nil {
-			return fmt.Errorf("lookup %s in source replica name registry: %v", tblName, err)
+		sourceReplicaTups := make([]sqlname.NameTuple, len(tableNameTups))
```

Comment:
Why are we calling lookup again here? I'm assuming it's because we now want to lookup in the context of SOURCE_REPLICA_DB_IMPORTER_ROLE? so that the state lookup happens properly? But role is tied to name registry object, right? 

Pls add a comment explaining